app/services/return-requirements/generate-return-version.service.js

```diff
@@ -21,7 +21,7 @@ const ReturnVersionModel = require('../../models/return-version.model.js')
  */
 async function go (sessionData, userId) {
   const returnVersion = await _generateReturnVersion(sessionData, userId)
-  const returnRequirements = await GenerateReturnVersionRequirementsService.go(sessionData.licence.id, sessionData.requirements)
+  const returnRequirements = await _generateReturnRequirements(sessionData)
```

Comment:
I think the top comment might need updating to explain that there is a no-returns required route, but other than that all good

Reply:
I've explained that on line 43 when it comes to generating the requirements. I can't see any need to add it to the top comment as well